gast_to_code/gast_to_code_main.py

```diff
@@ -8,79 +8,87 @@
 def list_helper(gast_list, out_lang, csv_delimiter = ", "):
     out = ""
     for i in range (0, len(gast_list)):
-        out += gast_router(gast_list[i], out_lang)
+        out += gast_to_node_router(gast_list[i], out_lang)
 
         if i< len(gast_list) - 1 : # don't add delimiter for last item
             out += csv_delimiter
     
     return out
-        
-def py_varAssign(gast):
-    value = gast_router(gast["varValue"], "py")
-    return gast_router(gast["varId"], "py") + " = " + value
 
-def binOp_helper(gast, out_lang):
-    op = " " + str(gast["op"]) + " "
-    left = gast_router(gast["left"], out_lang)
-    right = gast_router(gast["right"], out_lang)
+# assign helpers
+def gast_to_py_varAssign(gast):
+    value = gast_to_node_router(gast["varValue"], "py")
+    return gast_to_node_router(gast["varId"], "py") + " = " + value
 
-    return left + op + right
+def gast_to_js_varAssign(gast):
+    kind = gast["kind"]
+    varId = gast_to_node_router(gast["varId"], "js")
+    varValue = gast_to_node_router(gast["varValue"], "js")
+    return kind + " " + varId + " = " + varValue
 
-# py_specific_helpers
-def py_logStatement(gast):
-    arg_string = gast_router(gast["args"],"py")
-    return "print(" + arg_string + ")"
 
-def py_bool(gast):
-    if gast["value"] == 1:
-        return "True"
-    else:
-        return "False"
+# expression helpers
+def gast_to_py_functions(gast):
+    return gast_to_node_router(gast["value"], "py") + "(" + gast_to_node_router(gast["args"], "py") + ")"
 
-def py_boolOp(gast):
-    op = " and " if gast["op"] == "&&" else " or "
-    left = gast_router(gast["left"], "py")
-    right = gast_router(gast["right"], "py")
+def gast_to_js_functions(gast):
+    return gast_to_node_router(gast["value"], "js") + "(" + gast_to_node_router(gast["args"], "js") + ")"
 
-    return left + op + right
+def gast_to_py_attribute(gast):
+    return gast_to_node_router(gast["value"], "py") + "." + gast["id"] 
+
+def gast_to_js_attribute(gast):
+    return gast_to_node_router(gast["value"], "js") + "." + gast["id"] 
+
+def gast_to_py_logStatement(gast):
+    arg_string = gast_to_node_router(gast["args"],"py")
+    return "print(" + arg_string + ")"
 
-# js_specific_helpers
-def js_logStatement(gast):
-    arg_string = gast_router(gast["args"],"js")
+def gast_to_js_logStatement(gast):
```

Comment:
`gast_to_js_log_statement`... in general the snake casing vs camel casing is a bit inconsistent.

You could save that for another PR though.